src/iOS.Core/Services/BiometricService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Bit.Core.Abstractions;
+using Foundation;
+using LocalAuthentication;
+
+namespace Bit.iOS.Core.Services
+{
+    public class BiometricService : IBiometricService
+    {
+        private IStorageService _storageService;
+
+        public BiometricService(IStorageService storageService)
+        {
+            _storageService = storageService;
+        }
+
+        public bool SetupBiometric()
+        {
+            var state = getState();
+            return _storageService.SaveAsync("biometricState", state).IsCompletedSuccessfully;
```

Comment:
Also, I dont think our storage service supports storing NSData. Need to base64 it to a string.

Reply:
Hmm, yes. Checking the documentation it doesn't implement Serializeble.